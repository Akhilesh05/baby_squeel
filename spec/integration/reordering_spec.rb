require 'spec_helper'

describe '#reordering' do
  it 'orders using arel' do
    relation = Post.order(title: :desc).reordering do
      title.asc
    end

    expect(relation).to match_sql_snapshot
  end
end
