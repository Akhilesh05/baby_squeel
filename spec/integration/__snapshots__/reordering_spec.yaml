---
"#reordering orders using arel 1": SELECT "posts".* FROM "posts" ORDER BY "posts"."title"
  ASC
